tsconfig.json

```diff
@@ -1,17 +1,14 @@
 {
+  "extends": "@tsconfig/node18/tsconfig.json",
+
```

Comment:
This is actually probably the biggest change - this extends a TS config that works with Node 18. When we walk our node support we can change this and get the new configuration easily.

I removed all of the configuration that would re-specify what's already in the extension.